lms/static/js/student_account/views/account_settings_factory.js

```diff
@@ -191,6 +123,117 @@
+                        view: new AccountSettingsFieldViews.EmailFieldView({
+                            model: userAccountModel,
+                            title: gettext('Email Address'),
+                            valueAttribute: 'email',
+                            helpMessage: StringUtils.interpolate(
+                                gettext('The email address you use to sign in. Communications from {platform_name} and your courses are sent to this address.'),  // eslint-disable-line max-len
+                                {platform_name: platformName}
+                            ),
+                            persistChanges: true
+                        })
+                    },
+                    {
+                        view: new AccountSettingsFieldViews.PasswordFieldView({
+                            model: userAccountModel,
+                            title: gettext('Password'),
+                            screenReaderTitle: gettext('Reset Your Password'),
+                            valueAttribute: 'password',
+                            emailAttribute: 'email',
+                            passwordResetSupportUrl: passwordResetSupportUrl,
+                            linkTitle: gettext('Reset Your Password'),
+                            linkHref: fieldsData.password.url,
+                            helpMessage: StringUtils.interpolate(
+                                gettext('When you select "Reset Your Password", a message will be sent to the email address for your {platform_name} account. Click the link in the message to reset your password.'),  // eslint-disable-line max-len
+                                {platform_name: platformName}
+                            )
+                        })
+                    }
+                )
+            } else {
+                aboutSectionsData[0].fields.push(
+                    {
+                        view: new AccountSettingsFieldViews.ReadonlyFieldView({
+                            model: userAccountModel,
+                            title: gettext('Full Name'),
+                            valueAttribute: 'name',
+                            helpMessage: gettext(
+                                'The name that appears on your Statements of Accomplishment. Other learners never see your full name.'  // eslint-disable-line max-len
+                            )
+                        })
+                    },
+                    {
+                        view: new AccountSettingsFieldViews.ReadonlyFieldView({
+                            model: userAccountModel,
+                            title: gettext('Email Address'),
+                            valueAttribute: 'email',
+                            helpMessage: StringUtils.interpolate(
+                                gettext('The email address you use to sign in. Communications from {platform_name} and your courses are sent to this address.'),  // eslint-disable-line max-len
```

Comment:
Not using upstreams [`emailFieldView`](https://github.com/Stanford-Online/edx-platform/pull/844/files#diff-efea3005bf6db8bd17408044bebd80f7R52) as their updated help message gives an option to change their address which we don't want.

Reply:
https://github.com/Stanford-Online/edx-platform/blob/cc972e58a3bb6a4ae4684078cd0ece1d686a113b/lms/templates/student_account/account_settings.html#L41

Reply:
Sorry I should have been more clear, if you look at the help message in the `emailFieldView` non editable block, you will see that upstream has added an extra sentence "To change the email address, please contact {contact_email}" which I already confirmed with Monica yesterday that we don't want, hence why I went with not using their code at all to avoid more forking

Reply:
@kluo Got it.

In that case, can we selectively override it like mentioned for full name?

```js
fields[email_index] = {
    view: new AccountSettingsFieldViews.ReadonlyFieldView({
        // ...
    }),
};
```